picasa-starter.json

```diff
@@ -0,0 +1,25 @@
+{
+    "homepage": "https://sites.google.com/site/picasastartersite/",
+    "version": "2.0",
+    "description": "Picasa is a discontinued image organizer and image viewer for organizing and editing digital photos, plus an integrated photo-sharing website by Google",
+    "license": "Freeware",
+    "hash": "9188940457742451bfe100bf33df3ddf4ff0fa02ce2b242c599f0632a5a9cfbd",
+    "url": "https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/picasastarter/PicasaStarter%20V2.0.zip",
+    "bin": [
+      ["PicasaStarter.exe","picasaStarter"],
+      ["PicasaStarter.exe","picasa-starter"]
+    ],
+    "shortcuts": [
+        [
+            "PicasaStarter.exe",
+            "PicasaStarter"
+        ]
+    ],
+    "pre_install":[
+      "If(!(Test-Path \"$persist_dir\\PicasaStarterConfiguration.xml\")) { Set-Content \"$persist_dir\\PicasaStarterConfiguration.xml\" -Value $null }",
+      "If(!(Test-Path \"$persist_dir\\PicasaStarterSettings.xml\")) { Set-Content \"$persist_dir\\PicasaStarterSettings.xml\" -Value $null }",
+      "cmd /c mklink /H $dir\\PicasaStarterConfiguration.xml $persist_dir\\PicasaStarterConfiguration.xml",
```

Comment:
Why custom code persist links?? 
What's wrong with 

```json
"persist": ["PicasaStarterConfiguration.xml", "PicasaStarterSettings.xml"]
```

Reply:
PicasaStarter.exe assumes that the configuration files are in the same folder. I saw that the folders are seen locally to executable but the files not.

Reply:
> I saw that the folders are seen locally to executable but the files not.

I don't understand what you said 😶

persist_data function is doing completely same thing: https://github.com/lukesampson/scoop/blob/f284dba5d75ab336d968f6bad719be74261179d2/lib/install.ps1#L1252

Reply:
fixed. thank your for advices.